themes/origins/styles/component-modal.scss

```
@@ -0,0 +1,65 @@
+@import "mixins/breakpoints";
+
+yc-modal {
+  .modal-box {
+    z-index: 100;
+    position: fixed;
+    height: fit-content;
+    background-color: var(--color-base-white);
+  }
+
+  @media (min-width: 768px) {
+    .modal-box {
+      width: 500px;
+      inset: 50% 0 0 50%;
+      translate: -50% -50%;
+      border-radius: var(--radius-md);
+      transition: var(--transition-duration-slow)
+        cubic-bezier(0.175, 0.885, 0.32, 1.275);
```

The current animation spring causes the text to jitter when it arrives at the end of the animation, shall we stick with the classic ease-out, maybe even use a custom ease-out curve from [Emil Kowalski](https://x.com/emilkowalski_):
```css
cubic-bezier(0.215, 0.61, 0.355, 1); /* ease-out-cubic */
cubic-bezier(0.19, 1, 0.22, 1); /* ease-out-expo */
```

```suggestion
        cubic-bezier(0.175, 0.885, 0.32, 1.275);
```